mesos/src/main/scala/org/apache/spark/executor/MesosExecutorBackend.scala

```diff
@@ -75,7 +75,7 @@ private[spark] class MesosExecutorBackend
     val conf = new SparkConf(loadDefaults = true).setAll(properties)
     val port = conf.getInt("spark.executor.port", 0)
     val env = SparkEnv.createExecutorEnv(
-      conf, executorId, slaveInfo.getHostname, port, cpusPerTask, isLocal = false)
+      conf, executorId, slaveInfo.getHostname, port, cpusPerTask, None, isLocal = false)
```

Comment:
This means mesos is not suppported. Right? Could you throw an exception if IO encryption is enabled for mesos?